skipruntime-ts/ts/examples/utils.ts

```diff
@@ -75,10 +75,10 @@ class Player {
     }
   }
 
-  play(idx?: number) {
+  play(idx = 0) {
     let run = true;
-    if (idx ?? 0 > 0) {
-      run = this.start(idx!);
+    if (idx > 0) {
+      run = this.start(idx);
```

Comment:
Nit: reading a bit farther, I think we should use `number | undefined` or `number | null` rather than this 1-based index. But not important => skip

Reply:
Not disagreeing, but curious why you think that.

Reply:
0 is used to mean something else than an index, so we can use another type. `"kamoulox"` works as well